src/patternfly/components/Pagination/examples/pagination-top-disabled-example.hbs

```diff
@@ -0,0 +1,28 @@
+{{#> pagination}}
+  {{#> pagination-total-items}}
+    <b>1 - 10</b> of <b>37</b>
+  {{/pagination-total-items}}
+
+  {{> pagination-options-menu id="pagination-options-menu-top-disabled-example"  options-menu--IsText="true" options-menu-toggle--IsDisabled="true"}}
+
+  {{#> pagination-nav}}
+    {{#> button button--modifier="pf-m-plain pf-m-disabled" button--attribute='aria-label="Go to first page" aria-disabled="true"'}}
```

Comment:
Looks like in the pagination component, we're using `pf-m-disabled` wrong. We should be using `disabled` on `button`s and `pf-m-disabled` on `a` elements. So these buttons need to be updated to use `disabled`

Do you want to do that as part of this issue, or a follow up issue?

Reply:
I can do it in this issue